source/CargoHold.h

```diff
@@ -103,7 +103,11 @@ class CargoHold {
 	// so bad that it warrants a death sentence.
 	int IllegalCargoFine() const;
 	
-	
```

Comment:
Maintain the prescribed format for headers: 2 indented lines after a declaration when the next token is a visibility specifier.

Reply:
I just rewrote my changes to that file. Is this comment still relevant? I'm not sure what it refers to.

Reply:
corrected